src/material/core/option/option.ts

```diff
@@ -251,7 +251,6 @@ export class _MatOptionBase<T = any> implements FocusableOption, AfterViewChecke
   exportAs: 'matOption',
   host: {
     'role': 'option',
-    '[attr.tabindex]': '_getTabIndex()',
```

Comment:
Should we do this in the `legacy-option.ts` as well? The a11y tree there is identical and it would allow us to delete the `_getTabIndex` method.

Reply:
done ✔️